release-notes/release-notes58.md

```diff
@@ -25,7 +25,9 @@ keywords: releasenotes bugfixes
 
 The following new features were added for the {{site.productname}} 5.8 release.
 
-### Feature name
+### Add support for keys Home, End, PageUp and PageDown to escape inline elements
+
+A new set of keys has been added `Home`, `End`, `PageUp`, and `PageDown` to allow escape outside of the inline element scope (eg for links or comments).
 
```

Comment:
We need a "see" more link here that links to the inline_boundary setting.

Reply:
https://github.com/tinymce/tinymce-docs/pull/1946/files#diff-5c6bf60f64c3e3eec017e2e616a6fc0a8ce325aaea57e18f156c04eb1fcf5595L103 is an example of this

Reply:
updated